packages/grid/_modules_/grid/locales/esES.ts

```diff
@@ -0,0 +1,89 @@
+import { esES as esESCore } from '@material-ui/core/locale';
+import { GridLocaleText } from '../models/api/gridLocaleTextApi';
+import { getGridLocalization, Localization } from '../utils';
+
+const esESGrid: Partial<GridLocaleText> = {
+  // Root
+  rootGridLabel: 'grilla',
+  noRowsLabel: 'Sin filas',
+  errorOverlayDefaultLabel: 'Ha ocurrido un error.',
+
+  // Density selector toolbar button text
+  toolbarDensity: 'Densidad',
+  toolbarDensityLabel: 'Densidad',
+  toolbarDensityCompact: 'Compacta',
+  toolbarDensityStandard: 'Standard',
+  toolbarDensityComfortable: 'Comoda',
+
+  // Columns selector toolbar button text
+  toolbarColumns: 'Columnas',
+  toolbarColumnsLabel: 'Seleccionar columnas',
+
+  // Filters toolbar button text
+  toolbarFilters: 'Filtros',
+  toolbarFiltersLabel: 'Mostrar filtros',
+  toolbarFiltersTooltipHide: 'Ocultar filtros',
+  toolbarFiltersTooltipShow: 'Mostrar filtros',
+  toolbarFiltersTooltipActive: (count) =>
+    count !== 1 ? `${count} filtros activos` : `${count} filtro activo`,
+
+  // Export selector toolbar button text
+  toolbarExport: 'Exportar',
+  toolbarExportLabel: 'Exportar',
+  toolbarExportCSV: 'Descargar como CSV',
+
+  // Columns panel text
+  columnsPanelTextFieldLabel: 'Columna de búsqueda',
+  columnsPanelTextFieldPlaceholder: 'Título de columna',
+  columnsPanelDragIconLabel: 'Reorder columna',
+  columnsPanelShowAllButton: 'Mostrar todo',
+  columnsPanelHideAllButton: 'Ocultar todo',
+
+  // Filter panel text
+  filterPanelAddFilter: 'Agregar filtro',
+  filterPanelDeleteIconLabel: 'Borrar',
+  filterPanelOperators: 'Operadores',
+  filterPanelOperatorAnd: 'Y',
+  filterPanelOperatorOr: 'O',
+  filterPanelColumns: 'Columnas',
+  filterPanelInputLabel: 'Valor',
+  filterPanelInputPlaceholder: 'Valor de filtro',
+
+  // Filter operators text
+  filterOperatorContains: 'contiene',
+  filterOperatorEquals: 'es igual',
+  filterOperatorStartsWith: 'comienza con',
+  filterOperatorEndsWith: 'termina con',
+  filterOperatorIs: 'es',
+  filterOperatorNot: 'no es',
+  filterOperatorAfter: 'es posterior',
+  filterOperatorOnOrAfter: 'es en o posterior',
+  filterOperatorBefore: 'es anterior',
+  filterOperatorOnOrBefore: 'es en o anterior',
+
+  // Column menu text
+  columnMenuLabel: 'Menú',
+  columnMenuShowColumns: 'Mostrar columnas',
+  columnMenuFilter: 'Filtro',
+  columnMenuHideColumn: 'Ocultar',
+  columnMenuUnsort: 'Desordenar',
+  columnMenuSortAsc: 'Ordenar ASC',
+  columnMenuSortDesc: 'Ordenar DESC',
+
+  // Column header text
+  columnHeaderFiltersTooltipActive: (count) =>
+    count !== 1 ? `${count} filtros activos` : `${count} filtro activo`,
+  columnHeaderFiltersLabel: 'Mostrar filtros',
+  columnHeaderSortIconLabel: 'Ordenar',
+
+  // Rows selected footer text
+  footerRowSelected: (count) =>
+    count !== 1
```

Comment:
Are you sure? As far as I remember my Spanish lessons, it's
```suggestion
    count > 1
```

Reply:
You are right!

Reply:
I just copied from the `enUs` locale